### DISCUSSION TITLE
Enabling pre-commit doesn't file PRs

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#17655] [Request Help] [answered]
renovate.json
`{
    "extends": [
        "config:base",
        "workarounds:supportRedHatImageVersion",
        ":enablePreCommit"
    ],
    "enabledManagers": [
        "gitlabci",
        "pip_requirements",
        "dockerfile"
    ],
    "pre-commit": {
        "enabled": true
    }
}
`

**Comment by PhilipAbed** [comment, marked as answer]
I believe that the  `enabledManagers` is stronger than `preCommit: enabled`, 
also  `:enablePreCommit`, is already adding `preCommit: enabled`

try this:
```
{
  "extends": [
    "config:base",
    ":enablePreCommit",
    "workarounds:supportRedHatImageVersion"
  ],
  "enabledManagers": [
    "gitlabci",
    "pip_requirements",
    "dockerfile",
    "pre-commit"
  ]
}
```

**Reply by lmilbaum** [reply]
worked. thanks.
